pynitrokey/cli/fido2.py

```diff
@@ -236,16 +236,21 @@ def list_credentials(serial, pin):
             CredentialManagement.RESULT.RP_ID_HASH
         )
         local_print("-----------------------------------")
-        local_print(f"{reliable_party['name']}: ")
+        if "name" in reliable_party:
+            local_print(f"{reliable_party['name']}: ")
+        else:
+            local_print(f"{reliable_party['id']}: ")
```

Comment:
Could be better with:
```suggestion
        name = reliable_party.get('name', reliable_party['id'])
        local_print(f"{name}: ")
```